spec/sidekiq/form1010cg/submission_job_spec.rb

```diff
@@ -52,22 +52,89 @@
     let(:job) { described_class.new }
 
     context 'when there is a standarderror' do
-      it 'increments statsd' do
-        allow_any_instance_of(Form1010cg::Service).to receive(
-          :process_claim_v2!
-        ).and_raise(StandardError)
+      context 'caregiver1010 flipper enabled' do
+        before do
+          Flipper.enable(:caregiver1010)
+        end
+
+        after do
+          Flipper.disable(:caregiver1010)
+        end
```

Comment:
🎨 I've been using stubs for this - 
`expect(Flipper).to receive(enabled?).with(:caregiver1010).and_return(true)` / `false` to test the corresponding situations - any pros/cons between the approaches?

(personally I keep worrying about forgetting the `after`, but if they're next to each other like this that's less of a concern)